finsoc/app/assets/stylesheets/newsletter.scss

```diff
@@ -3,4 +3,48 @@
 		width: 100%;
 		height: 500px;
 	}
-}
\ No newline at end of file
+}
+/* Dropdown Button */
+.dropbtn {
+    background-color: #4CAF50;
+    color: white;
+    padding: 16px;
+    font-size: 16px;
+    border: none;
+    cursor: pointer;
+
+}
+
+/* Dropdown button on hover & focus */
+.dropbtn:hover, .dropbtn:focus {
```

Comment:
Make use of SASS syntax to make this more concise as you can place this inside of the .dropbtn block above i.e.
```
.dropbtn {
   ...
   &:hover {
      ...
   }
}
```